storybook/PagesModel.qml

```diff
@@ -0,0 +1,198 @@
+import QtQuick 2.14
```

Comment:
Imo `pages` dir is for pages and some components directly used by pages like model editors, this component is not a page. I left it on top because it's quite special, defines the content of the app. I wouldn't care about it too much now, it will be a subject for further changes soon.